libparsec/src/os.rs

```diff
@@ -0,0 +1,5 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+pub const fn get_os() -> &'static str {
+    std::env::consts::OS
```

Comment:
if we want an enum instead, we can use conditional compilation `#[cfg(target_os = "macos")]`